libs/domains/application/src/lib/slices/application.actions.ts

```diff
@@ -31,15 +32,11 @@ export const postApplicationActionsRedeploy = createAsyncThunk(
       }
 
       if (response.status === 202) {
-        // refetch deployments after update
-        if (data.withDeployments)
-          await dispatch(
-            fetchApplicationDeployments({
-              applicationId: data.applicationId,
-              serviceType: data.serviceType,
-              silently: true,
-            })
-          )
+        data.queryClient.invalidateQueries({
+          // NOTE: we don't care about the serviceType here because it's not related to cache
+          queryKey: queries.services.deploymentHistory({ serviceId: data.applicationId, serviceType: 'APPLICATION' })
```

Comment:
Even we don't have cache on the deployment side, it should be good to have a service Type, because the day we update the deployment History request with cache, we'll need this cache